test/hotspot/jtreg/runtime/ErrorHandling/StackWalkNativeToJava.java

```diff
@@ -70,6 +70,10 @@ public static void testStackWalkNativeToJavaNative(String... extraFlags) throws
         output.shouldNotContain("java.lang.RuntimeException: Reached statement after obj.wait()");
         output.shouldNotContain("[error occurred during error reporting (printing native stack");
         String[] res = output.getOutput().split("StackWalkNativeToJava\\$TestNativeToJavaNative\\.callNativeMethod\\(\\)V");
+        if (res.length - 1 != 2) {
```

Comment:
Pre-existing but this is a weird formulation - it should just be  `res.length != 3`.